src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,186 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.ui.parser.*;
```

Comment:
As per the Java coding standard, Imported classes should always be listed explicitly.